packages/components/checkbox/src/composables/use-checkbox.ts

```diff
@@ -0,0 +1,70 @@
+import { useFormItem, useFormItemInputId } from '@element-plus/hooks'
+import { isArray } from '@element-plus/utils'
+import {
+  useCheckboxDisabled,
+  useCheckboxEvent,
+  useCheckboxModel,
+  useCheckboxStatus,
+} from '../composables'
+
+import type { ComponentInternalInstance } from 'vue'
+import type { CheckboxProps } from '../checkbox'
+import type { CheckboxModel } from '../composables'
+
+const setStoreValue = (
+  props: CheckboxProps,
+  { model }: Partial<CheckboxModel>
```

Comment:
```suggestion
  { model }: Pick<CheckboxModel, 'model'>
```

I think the relevant types need to be optimized

```diff
- model!.value
+ model.value
```

Reply:
Thanks

Reply:
There are multiple files that have this problem

Reply:
Sorry for that, thanks for your guide. I have fixed these, please check.